ibis/backends/duckdb/__init__.py

```diff
@@ -443,6 +443,85 @@ def read_csv(
             con.exec_driver_sql(view)
         return self.table(table_name)
 
+    def _get_sqla_table(
+        self,
+        name: str,
+        schema: str | None = None,
+        database: str | None = None,
+        **_: Any,
+    ) -> sa.Table:
+        if schema is None:
+            schema = self.current_schema
+        *db, schema = schema.split(".")
+        db = "".join(db) or database
+        ident = ".".join(
```

Comment:
I would find a simple `if/else` with f-string more readable here.

Reply:
I don't think the `if`/`else` will be simple 😂

Reply:
Isn't this equivalent with 

```py
if db == self.current_database:
    ident = self._quote(schema)
else:
    ident = f"{self._quote(db)}.{self._quote(schema)}"
```
?